### PR TITLE
Remove useless debug folders

### PR DESCRIPTION
**Pull request by fivenp**
* Removing the `obj` + `bin` folders as they were added before the `.gitignore` was in place
* Adding `Debug/` as a default to be ignored....

I hope thats fine @RednibCoding?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
+.vs/
 bin/
-obj/
\ No newline at end of file
+obj/
+Debug/
```
Comment (RednibCoding) [inline .gitignore:4]: There is no such directory. Please remove the "Debug/" - entry

**Review by RednibCoding — DISMISSED** [review]

File: .gitignore (hunk 1 above)
Comment (RednibCoding) [reply .gitignore:4]: Also delete the .vs folder god damn it

Comment (fivenp) [reply .gitignore:4]: the Debug might be important for other subdirectories tho -> so `*/Debug/` will be ignored

**Comment by fivenp** [comment]
@RednibCoding we should be ready to go now? :)

**Pull request merged**